mr.ts

```diff
@@ -0,0 +1,113 @@
+// MR Title: Process user data, store in database, and count active users
+
+import sqlite3 from 'sqlite3';
+import { open } from 'sqlite';
+
+interface User {
+  id: number;
+  name: string | null;
+  email: string;
+  status: string | null;
+}
+
+// Function to process a list of user dictionaries and count active users
+function countActiveUsers(userList: User[]): number {
+  let activeUsers = 0;
+  for (const user of userList) {
+    if (user.status.toLowerCase() === 'active') {
```

Comment:
Need to check for null here.

```suggestion
if (user && user.status.toLowerCase() === 'active') {
```